File: ePadel/ePadel.Model/Reporti.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ePadel.Model
{
    public class Reporti
    {
        public int UkupnaZaradaTerena { get; set; } = 0;
        public int BrojRezervacijeTerena { get; set; } = 0;
        public int UkupanBrojKorisnika { get; set; } = 0;
        public int UkupnaZaradaSistema { get; set; } = 0;
        public int KorisnikId { get; set; }
    }
}

File: epadel_admin/lib/models/reporti.dart
import 'package:json_annotation/json_annotation.dart';

part 'reporti.g.dart';

/// dart run build_runner build

@JsonSerializable()
class Report {
  int? ukupnaZaradaTerena;
  int? brojRezervacijeTerena;
  int? ukupanBrojKorisnika;
  int? ukupnaZaradaSistema;
  int? korisnikId;


  Report({this.ukupnaZaradaTerena, this.brojRezervacijeTerena,this.ukupanBrojKorisnika});

  factory Report.fromJson(Map<String, dynamic> json) =>
      _$ReportFromJson(json);
  Map<String, dynamic> toJson() => _$ReportToJson(this);
}